packages/backend/src/apps/custom-api/actions/http-request/index.ts

```diff
@@ -79,6 +95,19 @@ const action: IRawAction = {
 
       $.setActionItem({ raw: { data: responseData } })
     } catch (err) {
+      if (
+        !(err instanceof HttpError) &&
```

Comment:
this will not pass, cos your error thrown will always be wrapped in a HttpError constructor.

Reply:
my bad i forgot to test back the original error wah *facepalm*, was trying to avoid the super odd case when someone throws another error with the same exact message, ok will remove this check, thanks!!!!